java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -0,0 +1,456 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands.servermodules;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.BaseClient;
+import glide.api.GlideClient;
+import glide.api.GlideClusterClient;
+import glide.api.models.ClusterValue;
+import glide.api.models.GlideString;
+import glide.api.models.commands.ConditionalChange;
+import glide.api.models.commands.json.JsonGetOptions;
+import glide.api.models.commands.json.JsonGetOptionsBinary;
+import glide.utils.ArgsBuilder;
+import glide.utils.ArrayTransformUtils;
+import java.util.concurrent.CompletableFuture;
+import lombok.NonNull;
+
+/** Module for JSON commands. */
+public class Json {
+
+    public static final String JSON_PREFIX = "JSON.";
+    public static final String JSON_SET = JSON_PREFIX + "SET";
+    public static final String JSON_GET = JSON_PREFIX + "GET";
```

Comment:
Could be private.

Reply:
let's leave it with future refactoring, a lot of other Options classes have the same.